syntax.php

```diff
@@ -184,7 +184,7 @@ function render($format, Doku_Renderer $renderer, $data) {
         $data += $this->query($data['id']);
 
         // link title
-        $link = 'issue #'. $data['id'];
+        $link = $this->getLang('issue') . ' #'. $data['id'];
```

Comment:
basic translation principles, word order in sentence not the same in all language.

must include issue id in translation as well, and use `sprintf %s`.